### ISSUE TITLE
TweakScale not installed on wrong directory

### ISSUE THREAD

**Issue by kolAflash** [Biotronic/TweakScale#39]
When starting KSP there's a message:

> Houston, we have a problem!
> 
> Unfortunately TweakScale is installed on the wrong directory!
> 
> It should be installed on /home/user/opt/Steam/steamapps/common/Kerbal Space Program/GameData/TweakScale/ but it's currently installed on /home/user/GamesStuff/KerbalSpaceProgram/GameData/TweakScale/ ! Delete the latter and be sure to install TweakScale on the former.
> [...]

`/home/user/GamesStuff/` is where I keep stuff like savegames, mods, ... for different games, so I can easily backup it.
And `/home/user/opt/Steam/steamapps/common/Kerbal Space Program/GameData/TweakScale/` is symlinked to `/home/user/GamesStuff/KerbalSpaceProgram/GameData/TweakScale/`, so it's actually in the right place.

**Comment by Biotronic** [comment]
It's been six years (damn, I'm getting old :p) since I ceded control of TweakScale to Pellinor, who has since given it to [lisias](https://github.com/net-lisias-ksp/TweakScale). I suggest you post your question [there](https://github.com/net-lisias-ksp/TweakScale) instead. Sorry I'm not of much help, but I'm not maintaining TweakScale, and not really even playing KSP these days.

I'm unable to ignore an issue though, so I did some searching. [This issue](https://github.com/net-lisias-ksp/KSPAPIExtensions/issues/6) from KSPAPIExtensions seems very related, and might help you solve your problem.
